tests/client/amo/components/TestSearchForm.js

```diff
@@ -0,0 +1,111 @@
+import React from 'react';
+import { Simulate, renderIntoDocument } from 'react-addons-test-utils';
+
+import * as actions from 'core/actions';
+import * as coreApi from 'core/api';
+import { SearchFormBase, mapDispatchToProps, mapStateToProps }
+  from 'amo/components/SearchForm';
```

Comment:
I find this formatting a little out of place. We've normally been breaking within the parens:

```js
import {
  SearchFormBase,
  mapDispatchToProps,
  mapStateToProps,
}  from 'amo/components/SearchForm';
// or maybe
import {
  SearchFormBase, mapDispatchToProps, mapStateToProps,
}  from 'amo/components/SearchForm';